docs/storage/storage.md

```diff
@@ -190,6 +190,10 @@ XOSTOR is coming soon in XCP-ng. Hang on!
 
 ### ZFS
 
+:::warning
+**ZFS is offered as-is** and does not come with official support.
```

Comment:
What's offered as-is is a storage driver that can use a ZFS mount point, not ZFS itself (well, actually we also offer RPM packages for ZFS itself, but that's not my point :D).

What about "ZFS storage integration is offered as-is..."? (CC @Fohdeesha)

Reply:
Agreed, and "ZFS storage integration is offered as-is" is the phrasing I would put here as well.

Reply:
Fixed by https://github.com/xcp-ng/xcp-ng-org/pull/300/commits/04501a12e5566c79a6af708273078faefe3a000e